tekton-catalog/pipeline-loops/examples/loop-example-numeric-param.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: custom.tekton.dev/v1alpha1
+kind: PipelineLoop
+metadata:
+  name: pipelineloop
+spec:
+  # IterationNumberParam points to the param that stores the current loop item. Similar to IterateNumeric
+  # for numeric loop range.
+  iterationNumberParam: current-message
```

Comment:
`current-message` is a very misleading name here. It's usage, `echo "$(params.message) $(params.current-message)"`, doesn't make it any better.

Maybe `message-index` would do better? And `echo "$(params.message-index)) $(params.message)"`, so that it's shown as:
```
1) I am the first one
2) I am the second one
3) I am the third one
```